packages/dialect-react-ui/components/Notifications/Header.tsx

```diff
@@ -58,7 +58,7 @@ function Header(props: HeaderProps) {
     <>
       <div className="dt-flex">
         <SettingsButton />
-        <div className={clsx(!props.settingsOnly && 'sm:dt-hidden', 'dt-ml-3')}>
```

Comment:
This seems like a breaking change. Is this necessary to remove?

Reply:
added explicit prop to configure close button visibility